connector/pom.xml

```diff
@@ -69,6 +69,11 @@
             <artifactId>pixels-invoker-lambda</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>io.pixelsdb</groupId>
+            <artifactId>pixels-invoker-vhive</artifactId>
```

Comment:
Remove this dependency. Multiple invoker dependencies will overwrite the invokers for a worker type.

To solve this problem, we need to modify the InvokerFactory in pixels-common and allow it to load multiple invokers for the same worker type into invokerMap.